### PR TITLE
Added redis deployment config to template yaml

### PR DESCRIPTION
**Pull request by caseyfw**

### CODE REVIEW

File: shepherd-openshift.yml [hunk 1]
```diff
@@ -10,18 +10,20 @@ metadata:
     openshift.io/display-name: Shepherd
     description: Provide a template that deploys an instance of the shepherd distribution.
     tags: php,drupal
-    template.openshift.io/documentation-url: https://gitlab.adelaide.edu.au/web-team/shepherd
+    template.openshift.io/documentation-url: 'https://gitlab.adelaide.edu.au/web-team/shepherd'
```
Comment (caseyfw) [inline shepherd-openshift.yml]: So apparently in yaml any value with a colon in it should be wrapped in single quotes.

File: shepherd-openshift.yml [hunk 2]
```diff
@@ -44,14 +46,35 @@ objects:
...
+    name: shepherd-redis-is
+  spec:
+    lookupPolicy:
+      local: false
+    tags:
+      - annotations:
+          openshift.io/imported-from: 'redis:alpine'
+        from:
+          kind: DockerImage
+          name: 'redis:alpine'
```
Comment (caseyfw) [inline shepherd-openshift.yml]: @singularo in your shp_redis_support module this line is more like `name: docker.io/redis:alpine`. After some testing, the above is what openshift does when you just put a regular image ref into the "add to project" box.

File: shepherd-openshift.yml (hunk 1 above)
Comment (singularo) [reply shepherd-openshift.yml]: Ahah.. Its a trap.

File: shepherd-openshift.yml [hunk 3]
```diff
@@ -144,7 +169,7 @@ objects:
             value: /code/web/sites/default/files
           - name: SHEPHERD_INSTALL_PROFILE
             value: ${SHEPHERD_INSTALL_PROFILE}
-          image: ' '
+          image: 'shepherd-web-is:latest'
```
Comment (caseyfw) [inline shepherd-openshift.yml]: I'm not even sure why the image ref of `' '` worked originally.

File: shepherd-openshift.yml [hunk 4]
```diff
@@ -186,22 +211,24 @@ objects:
       port: 3306
       targetPort: 3306
     selector:
-      name: shepherd-db
+      name: shepherd-db-dc
```
Comment (caseyfw) [inline shepherd-openshift.yml]: Now with more Smurfing :tm:

File: shepherd-openshift.yml (hunk 1 above)
Comment (singularo) [reply shepherd-openshift.yml]: Err @caseyfw this content should point to this repo now. ;-)

File: shepherd-openshift.yml [hunk 5]
```diff
@@ -211,8 +238,8 @@ objects:
             value: ${DATABASE_PASSWORD}
           - name: MYSQL_USER
             value: ${DATABASE_USER}
-          image: ''
-          name: mysql
+          image: 'openshift/mariadb:latest'
+          name: shepherd-db
```
Comment (caseyfw) [inline shepherd-openshift.yml]: This part looks really weird in contrast to the others in relation to the ImageChange trigger spec below. Here the image is specified as `openshift/mariadb:latest` but below it's called `mariadb:latest` and the namespace is in a separate key. Weird.

File: shepherd-openshift.yml (hunk 2 above)
Comment (singularo) [reply shepherd-openshift.yml]: Ahah. Will fix there in another MR, thanks for that spotto.

**Pull request merged**